doc/release-notes/sourmash-4.0.md

```diff
@@ -0,0 +1,71 @@
+# sourmash v4.0 release notes
+
+```{contents}
+   :depth: 2
+```
+
+We are pleased to announce release 4.0 of sourmash! This release
+contains many feature improvements and new functionality, as well as
+many breaking changes with sourmash 2.x and 3.x.
+
+Please see
+[our migration guide](../support.md#migrating-from-sourmash-v3-x-to-sourmash-v4-x)
+for guidance on updating to sourmash v4, and post questions about
+migrating to sourmash 4.0 in the
+[sourmash issue tracker](https://github.com/dib-lab/sourmash/issues/new).
+
+## Major changes for 4.0
+
+### New or changed behavior
+
+* default SBT storage is now .sbt.zip (#1174, #1170)
+* add `sourmash sketch` command for creating signatures (#1159)
+* protein ksizes in MinHash are now divided by 3, except in `sourmash compute` (#1277)
+* refactor MinHash API and implementation: add, iadd, merge, hashes, and max_hash (#1282, #1154, #1139, #1301)
+* add HyperLogLog implementation (#1223)
```

Comment:
add HyperLogLog implementation of what :) 
May not be important to add that detail here, but also could be nice

Reply:
I thought about this before, and decided not to. They can google or look at #1123 if they want - right now it's not particularly accessible and has no real impact on sourmash functionality, IIRC. But good catch :)